limiinal/limiinal
# Backend and  frontend linking.

## **Description**
* Implemented the backend allow client to connect to peer via command line arguments (temporary) 
* Link backend to frontend. (run the backend as [Task](https://docs.rs/iced/latest/iced/task/struct.Task.html) )


## **Checklist**
- [x] All Rust code is formatted using `cargo fmt`.
- [x] Documentation has been updated as necessary (`README.md`, code comments, etc.).

## **Testing Instructions**
Run the program with `RUST_LOG=info` should see the `Backend is running` message.


## **Related Issues**
The backend is currently panicking, it will be resolved soon.

## **Types of Changes**
